gensim/models/lsimodel.py

```diff
@@ -427,15 +430,20 @@ def __init__(self, corpus=None, num_topics=200, id2word=None, chunksize=20000,
                 )
             try:
                 import Pyro4
-                dispatcher = Pyro4.Proxy('PYRONAME:gensim.lsi_dispatcher')
-                logger.debug("looking for dispatcher at %s", str(dispatcher._pyroUri))
-                dispatcher.initialize(
-                    id2word=self.id2word, num_topics=num_topics, chunksize=chunksize, decay=decay,
-                    power_iters=self.power_iters, extra_samples=self.extra_samples, distributed=False, onepass=onepass
-                )
-                self.dispatcher = dispatcher
-                self.numworkers = len(dispatcher.getworkers())
-                logger.info("using distributed version with %i workers", self.numworkers)
+                if ns_conf is None:
+                    ns_conf = {}
+
+                with utils.getNS(**ns_conf) as ns:
+                    from gensim.models.lsi_dispatcher import LSI_DISPATCHER_PREFIX
```

Comment:
Is there a reason to keep this import here in a try-catch? If not, please move it to the module scope.